src/exceptions/middleware-error-handler.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response, NextFunction } from 'express';
```

Comment:
I have questions regarding folder and file names. Each function you pass to express or routers is middleware. For me this folder only appropriate for storing error classes.

My opinion is to place this file as error controller but without extending Abstract controller, then directly launch it in app.